lesson2/app/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "user_with_pet_app.js",
+  "type": "module",
+  "scripts": {
+    "start": "node user_with_pet_app.js"
+  },
+  "author": "",
+  "license": "ISC"
+}
```

Comment:
Add a blank line in the end of the file.
*Note: always end your text files with a blank line.*
*FYI: [why should text files end with a newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).*